pontoon/base/models.py

```diff
@@ -2802,7 +2891,7 @@ def save(self, *args, **kwargs):
         translatedresource, _ = TranslatedResource.objects.get_or_create(
             resource=self.entity.resource, locale=self.locale
         )
-        translatedresource.calculate_stats()
+        # translatedresource.calculate_stats()
```

Comment:
You can remove this instead of commenting it out and also the block above, because we don't need `translatedresource` anymore.